### ISSUE TITLE
How export complete configuration (all profiles and apps settings)

### ISSUE THREAD

**Issue by xes** [schwabe/ics-openvpn#1392]
Is there a way to export all the configuration / all the profiles with included enabled apps?
(not only the openvpn conf but also the ics-openvpn settings for it)
Obviously the target is to re-import it or the prepare a complete profile for a new device to setup.

If not available, do you think this could be a feature that could be planned for the future? 

Thank you in advance!

**Comment by schwabe** [comment]
No, currently there is nothing like that. In the past adb backup/adb restore worked reasonable well enough. And currently I have no plans implementing it.

**Comment by hmueller01** [comment]
I like to switch from Google play to F-droid. As there are different signings I need to deinstall openVPN first and reinstall it from F-droid. Is it possible to restore the settings with `adb backup` and `adb restore`?

**Comment by schwabe** [comment]
No idea to be honest, never tried.

**Comment by hmueller01** [comment]
Thanks for the quick reply. I tried adb backup but I only get 47 bytes. Looks like an adb problem (reported by others). No further inverstigation needed. Just an info. Tx for the great work!

**Comment by Human** [comment]
Titanium Backup isn't working here, either. I even tried `tar`ring up the `de.blinkt.openvpn` directory and restoring it. All the files are there, owned by the new device's user for the openvpn app. Running the app tells me "No VPN profiles defined" :(

**Comment by Human** [comment]
In my case, the option to encrypt the profiles was turned on. I will try turning that off on my old device to see if I can port to my new device.

**Comment by Human** [comment]
I turned the profile encryption option off, but I had to make a change to the profile before I ended up with an unencrypted profile. I had to restart my new phone before the profile started working, but it works now!

**Comment by surlyhacker** [comment]
> I turned the profile encryption option off, but I had to make a change to the profile before I ended up with an unencrypted profile. I had to restart my new phone before the profile started working, but it works now!

Thank you @Human !  That's very helpful
